packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -311,6 +311,11 @@ export const MOCK_CHECKOUT_TOKEN = {
   token: 'EC-8NC18566WJ1581100',
 };
 
+export const MOCK_PAYPAL_SUBSCRIPTION_RESULT = {
+  sourceCountry: 'Equatorial Kundu',
```

Comment:
Is this also made up or does PayPal return a full country name instead of a two character code country code?

Reply:
This is a [two-character country code per the fxa-auth-server validator](https://github.com/mozilla/fxa/blob/021fd5d4485aebf174fe668e3d6db15c6ae7cf1f/packages/fxa-auth-server/lib/routes/validators.js#L288-L291).

My client side TS only asserts it's a string. Looks like fxa-auth-server uses `hapi` to add more specificity here, but I don't see similar validation on the client in `apiClient.ts`. I can change this string in my test to be consistent however.

Reply:
Would that suffice here, or is there more I should do @chenba ?

Reply:
Oh it wasn't a request for a change.  I was just curious.  I wasn't sure if you copied that from a (sandbox) response or something.

Reply:
I just put a made up country from [The West Wing](https://en.wikipedia.org/wiki/The_West_Wing). 😆